chart/values.yaml

```diff
@@ -476,6 +476,20 @@ datasetUpload:
           cpu: 200m
           memory: 128Mi
 
+tusd:
+  replicas: 1
+  image:
+    repository: tusproject/tusd
+    tag: v1.4.0
```

Comment:
* it should cover in the images.yaml for airgap in ce and `images.yaml.gotmpl` in ee

Reply:
Thanks!